revised/beamswap-proposal.md

```diff
@@ -0,0 +1,330 @@
+
+
+Onboarding new users from different corners of the Web3 space and beyond proves to be a challenging but not impossible task. To deliver the latest DeFi services and features to different communities on various networks, the Beamswap DEX integrates Multichain, the bridge with [the highest TVL](https://defillama.com/protocols/Bridge) across all EVM networks.
+
+For users to have more authority and sovereignty, they should be able to choose their preferred stablecoin among all the supported coins on the Moonbeam Network. As a Moonbeam-native DEX and perpetual exchange protocol harnessing the power of Moonbeam’s connected contracts, Beamswap and Beamex aim to provide direct access to Multichain stablecoins together with Polkadot-native stablecoins.
+
+### TVL and APY estimates
+
+Token prices used for calculations: (10th March 2023)
+* GLMR = USD 0.35
+* GLINT = USD 0.00092
+
+Beamswap reserves the right to discontinue or pause GLMR emissions in case of extreme events on the Moonbeam Networks, such as, but not limited to:
+
+* Moonbeam Foundation’s request to stop/pause the program
+* Significant price manipulation on one of the assets (>70% price change)
+* Bridge exploit (Nomad, aUSD scenario)
+* Smart contract risk (unverified contracts that might affect the Beamswap platform)
+* Auto-compounder exploit (LP migration)
+* Network downtime (upgrades, pauses, etc.)
+
+In case of other detrimental events, Beamswap would also halt rewards and return the remaining grant amount to the Moonbeam Treasury.
+
+Expected TVL is an estimate based on the current token prices and network activities.
+
+### Phase 1
+
+In the first four weeks, a liquidity-sufficient order book would be developed for assets included in the Beamex Perpetual Index token, BLP. With the launch of Beamswap v3 concentrated liquidity AMM and proposed TVL, ~95% of all Moonbeam-based transactions to date can be covered.
+
+Through the use of XCM transfer and connected contracts, a stable pool (3pool) consisting of USDTxc, USDCaxl, and USDCmulti would be deployed.
+
+![|602x339](upload://stGnLWpgEEQMwhCLHXfGMDckMSP.jpeg)
+
+Achieving the forecasted TVL would lead to more daily volume on the Beamswap DEX while increasing the Moonbeam Network key metrics in active users, transactions, and total value locked.
+
+Expected APY and TVL should be met within the first week of the Beamswap v3 launch.
+
+The success of Phase 1 and Beamswap’s capacity to proceed with Phase 2 would be measured by unique user growth, transaction size growth, TVL, interactions with the exchange’s smart contracts, and more.
+
+### Phase 2
+
+Following the successful Phase 1 of building sufficient liquidity, the Beamex perpetual DEX, the first perpetual exchange on Polkadot Network, is planned to launch.
+
+Using Beamswap Flywheel, users will be able to benefit from Moonbeam and Polkadot capabilities to open long or short positions on Beamex-supported assets.
+
+![|602x339](upload://4UU32xt3zIprYMzEIq4fgiebraE.jpeg)
```

Comment:
Can you also upload the images to the revised/assets/beamswap folder? Otherwise we'll end up with some empty placeholders.